go.mod

```diff
@@ -1,67 +1,66 @@
 module github.com/goware/cachestore
 
-go 1.21
+go 1.23.0
```

Comment:
FYI, this is bumping the minimal required version of Go that needs to be used with this package.

Consider reverting back to go 1.21, unless necessary.

Reply:
Understood! I have managed to keep the same version